### PR TITLE
docs profile user

### PR DESCRIPTION
**Pull request by mvn-nale-dn**
## Description
Docs profile user
## Reference Requirements
- Task ID: #123

## Reference Isuees
#{issue_id}

## Migrations
 NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Comment by at-haytran** [comment]
LGTM

File: docs/api-references/profile.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+| /api/users/profile | Get profile of user |
+
+```json
+{
+    "data": {
+        "id": 1,
+        "email": "jhayes@example.com",
+        "name": "Mr. Fabian Rippin PhD",
+        "identity_number": "375699562",
+        "avatar": "default-user.png",
```
Comment (at-vietpham) [inline docs/api-references/profile.md]: Hình này phải tra về full URL luôn.

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by at-haytran** [comment]
LGTM

**Comment by at-vietpham** [comment]
LGTM 🎲

**Pull request merged**